common/store/configAndTokens.ts

```diff
@@ -135,6 +136,13 @@ function getSavedSelectedNode(
     ? staticNodes[savedNodeId]
     : customNodes[savedNodeId];
 
+  if (nodeConfigExists) {
+    if (isAutoNode(savedNodeId)) {
+      shepherd.switchNetworks(nodeConfigExists.network);
+    } else {
+      shepherd.manual(savedNodeId, false);
+    }
+  }
```

Comment:
We might do well to implement something in here that sets users to auto who have previously loaded the beta. I was defaulted to the MyCrypto node, because I had a stored config already.

Reply:
I think we should first implement a versioning spec for local storage so we can properly invalidate data for situations like this. Since this case pops up often enough to warrant it imo